src/autoscaler/scalingengine/server/scaling_handler.go

```diff
@@ -19,17 +19,22 @@ type ScalingHandler struct {
 	logger          lager.Logger
 	scalingEngineDB db.ScalingEngineDB
 	scalingEngine   scalingengine.ScalingEngine
+	health          models.Health
 }
 
-func NewScalingHandler(logger lager.Logger, scalingEngineDB db.ScalingEngineDB, scalingEngine scalingengine.ScalingEngine) *ScalingHandler {
+func NewScalingHandler(logger lager.Logger, scalingEngineDB db.ScalingEngineDB, scalingEngine scalingengine.ScalingEngine, health models.Health) *ScalingHandler {
 	return &ScalingHandler{
 		logger:          logger.Session("scaling-handler"),
 		scalingEngineDB: scalingEngineDB,
 		scalingEngine:   scalingEngine,
+		health:          health,
 	}
 }
 
 func (h *ScalingHandler) Scale(w http.ResponseWriter, r *http.Request, vars map[string]string) {
+	h.health.Inc("concurrentHTTPReq")
```

Comment:
do we want to collect separately for request coming from eventgen and scheduler?